frontend/src/app/app.component.ts

```diff
@@ -2,7 +2,7 @@ import { Component, OnInit } from '@angular/core';
 import { FileSelectDirective, FileDropDirective, FileUploader } from '../../node_modules/ng2-file-upload/ng2-file-upload';
 import 'rxjs/Rx' ;
 
-const URL  = 'https://qpp.cms.gov/api/submissions/converter';
+const URL  = 'http://localhost:8080/';
```

Comment:
is this accurate @saquino0827

Reply:
For Local host yes, I feel like this was pointed to dev-pre at some point. Let me check.

Reply:
The dev-pre URL is `https://preview.qpp.cms.gov/api/submissions/converter`. I believe if we wanted to test this in dev, we may have to get our s3 bucket configured to work through the VPN. And for IMP we would need to include the cookie. Localhost lets you deploy it locally and test.